### DISCUSSION TITLE
Result png contains too much redundant information

### DISCUSSION THREAD

**Discussion by danjohles** [kinow/protobuf-uml-diagram#75] [General]
I am not sure if this should be an issue or it's just a setting I am not aware of

But the resulting images do not look like this one: ![this](https://raw.githubusercontent.com/kinow/protobuf-uml-diagram/master/example-output.png) 
The title is the title and attributes being listed without the title as a prefix
They do look like the other example: ![other](https://github.com/kinow/protobuf-uml-diagram/blob/master/.github/docs/issue_10_pb2.png?raw=true) 
Is there a way to get the first image out of a proto instead of the second?

**Comment by kinow** [comment]
Hi @danjohles , sorry, I didn't understand your question. I think you have a protobuf schema that you are trying to plot, but the image is not displaying something you wanted? Could you be more specific, or try to re-word the issue in another manner, please? Thanks.

**Reply by danjohles** [reply]
Thanks for the reply!

I'm sorry if I wasn't clear enough.
The first image for example , the table  called PbCondition
It has properties like
+ task_proxy:string
+ expr_alias:string 
Etc.

This is the behavior I would expect according to this Image
The result in the second would be 
PbCondition as the title and property names like
+ PbCondition.task_proxy:string
+ PbCondition.expr_alias:string

I.e every property inside the UML has the table name as a prefix. Is there some option that could be set to get the behavior of the first image? Meaning the properties without the table label as a prefix. I hope this was a little clearer.
Greetings

**Reply by kinow** [reply]
Hi @danjohles , thanks for the explanation. I think I understand it now.

See #78 and #79 for the fix. TL;DR in `>0.13` you should have a command line parameter `full_names=true|false|<Default=true>`. Just set it to `false` and you should get simpler names in the class fields.

See issue #10 for more about why we changed that. It would be grand if you could test 0.13 (being uploaded now) and let me know later if that worked as expected :+1: 

Cheers, Bruno

**Reply by kinow** [reply]
0.13 is being prepared now, it will take some minutes (10?) until it's on PyPI

**Reply by danjohles** [reply]
Upgraded and tested it
works great
thanks!

**Reply by kinow** [reply]
Thanks for confirming it!
